### ISSUE TITLE
FTB-AUTH-000006

### ISSUE THREAD

**Issue by n1b0r63** [FTBTeam/FTB-App#983]
### What Operating System

Windows 10

### Debug Code

FTB-DBGXIWOXAYIBA

### Describe the bug

Fail to launch because faultly can't find the game on the account

### Steps to reproduce

Installed the app, installed direwolf20 1.20 modpack, try to launch it, connect correctly on my browser to my microsoft account but it show me this error.

I'm 100% sure I have the game on the account I connected to, I check on minecraft.net and I even managed to launch correctly using the curseforge launcher using the exact same credentials on microsoft (but there is not the latest direwolf20 pack on curseforge).

What I tried:
- Change the browser, run it with the link, etc...
- Uninstalled and reinstall the FTB app
- Run it with and without the administrator rights
- Restart the PC

### Expected behaviour

The game starting correctly

### Screenshots

![image](https://github.com/FTBTeam/FTB-App/assets/37269210/dabc4c41-d153-496a-b4be-021a32fb76fe)

### Additional information

I also got other errors along my tests, all starting with FTB-AUTH-00000, like the 1 and the 5, but I'm not 100% sure what test I've done to get those, the 6 is the main one.

**Comment by n1b0r63** [comment]
Ok I'm not sure what happened, after lots of tests I re-bought the game on the same account and apparently it wasn't considered active on it due to all their shenanigans during migration probably. Still a complete mystery for me, I'm 100% sure it works on curseforge (I tested like 10 times during my tests) with the exact same account (I only have one) with the same ID/pass (again, I only got one account and copied pasted the same ids). I used to have 2 minecraft accounts, I tested the 2 old auth it didn't worked (as expected) so it's not what connected curseforge, and I'm sure I migrated one of these account on microsoft (and seeing that it worked with curseforge I'm sure this microsoft account somehow already had minecraft).
Maybe it's like the minecraft version that was not considered the same (like bedrock or something ?) or it depends of how the API to asked if there is minecraft is not the same ? Anyhow if someone has any idea what is happening I would be glad to know. It's already the third time I buy it because of these kind of problems :(
